### DISCUSSION TITLE
swiftformat をローカルインストールしたい

### DISCUSSION THREAD

**Discussion by uhooi** [YusukeHosonuma/Swift-Evolution-Browser#41] [Ideas]
現状だと swiftformat を Homebrew でインストールしていますが、グローバルインストールされてしまいます。
できる限りグローバル環境を汚したくないため、Swift 製の CLI ツールは [Mint](https://github.com/yonaskolb/Mint) or SwiftPM で管理するのがいいと思います。
（Mint 自体はグローバルインストールする必要がありますが）

もしくは Swift 5.6 で追加された SwiftPM のプラグインを使うのもよさそうです。

**Comment by YusukeHosonuma** [comment]
@uhooi 
ご意見ありがとうございます 🙏 
確かにグローバルインストールが嫌だという人は一定数居ますよね...よく分かります。

> Swift 5.6 で追加された SwiftPM のプラグイン

こちらも気になっているので、前向きに検討したいと思います 👍 

P.S.
書かれていた記事を読ませていただきました 🙏 
https://qiita.com/uhooi/items/6a41a623b13f6ef4ddf0

**Reply by YusukeHosonuma** [reply]
@uhooi 
#52 にて Mint 対応させていただきました 🙏 
SPM の Command Plugin の方も余力があれば試してみたいなぁと思っています。

**Reply by uhooi** [reply]
おお…さっそく！
ありがとうございます 🙏 
記事も読んでいただき嬉しいです…！
